Cargo.toml

```diff
@@ -1,3 +1,9 @@
+[package]
+name = "toplevel"
+version = "0.0.0"
```

Comment:
What is this for?

Reply:
See: https://github.com/holochain/holochain/pull/662/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R1

More details: `cargo release --package hdk` in workspace with no toplevel crate would publish all crates which we don't always want. We would have to use `--exclude ...` for all non-desired crates, but I find that cumbersome as it would need an exhaustive list. Therefore I chose this workaround so we can have an inclusive method.

Reply:
Yep this completely breaks my ide and ability to run tests. I think we need a different workaround

Reply:
I removed it. We'll be able to do without it somehow.